client/src/components/Testimonial.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styles from "../styles/Testimonials.css"
```

Comment:
Looks Like something is wrong with this statement.

Reply:
I'll take a look at it